bindata/network/multus/multus.yaml

```diff
@@ -455,3 +455,42 @@ spec:
           configMap:
             name: cni-binary-copy-script
             defaultMode: 0744
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: ip-reconciler
+  namespace: openshift-multus
+  labels:
+    tier: node
+    app: whereabouts
+spec:
+  schedule: "*/5 * * * *"
```

Comment:
Can you add a comment before this explaining what it means? I always forget which cronjob field is which...